server/server.go

```diff
@@ -328,6 +332,9 @@ func NewServer(config *Config) (*Server, error) {
 		return nil, err
 	}
 
+	// here we can provide some other configuration
+	m.gasHelper = gasprice.NewGasHelper(gasprice.DefaultGasHelperConfig, m.blockchain)
```

Comment:
Should we expose gas helper config to server configuration as well? Does it make sense to have different configuration parameters per server instance. If it does, I don't see how we could provide different parameters, other than changing the code. 

On the other hand if those params should be network wide, then obviously we should put it as a part of on-chain governance.

To me, it makes sense either way, as I couldn't find in the specification is it expected that every single node in network must calculate the same value.

WDYT?

Reply:
They can't be per server. It's a network wide configuration, probably provided through genesis, and on-chain governance. For now, I've left it to default values that `geth` uses, but in the future we should definitely provide a means to users to configure this through governance.